examples/glusterfs/README.md

```diff
@@ -0,0 +1,42 @@
+## Glusterfs
+
+[Glusterfs](http://www.gluster.org) is an open source scale-out filesystem. These examples provide information about how to allow Docker containers use Glusterfs volumes.
+
+The example consists of a pod that runs on hosts that install Glusterfs client package.
+
+### Prerequisites
+
+Install Glusterfs client package on the hosts.
+
+### Create a POD
+
+The following *volume* spec illustrates a sample configuration.
+
+```js
+{
+     "name": "glusterfsvol",
+     "glusterfs": {
+        "hosts": "glusterfs-cluster",
+        "path": "kube_vol",
+        "mountOptions": "ro",
+        "helper": ""
+    }
+}
+```
+
+The parameters are explained as the followings. **hosts** is endpoint name that defines Gluster service. **kubelet** is optimized to avoid mount storm, it will randomly pick one from the hosts to mount. If this host is unresponsive, the next host in the array is automatically selected. **path** is the Glusterfs volume name. **mountOption** is the mount time options, it can cotains standard mount time options such as *ro* (readonly). **helper** can be a command that can be executed prior to mounting the filesystem.
+
+Detailed POD information can be found at [v1beta3/](v1beta3/)
```

Comment:
You need to tell the user to edit glusterfs-service.json and put in the right IPs.